test/test_puma_server.rb

```diff
@@ -374,7 +374,7 @@ def test_status_hook_fires_when_server_changes_states
   end
 
   def test_timeout_in_data_phase
-    @server.first_data_timeout = 2
+    @server.first_data_timeout = 0.01
```

Comment:
Why was this changed?

Reply:
This was to just help reproduce #2282 more quickly, but doesn't need to be committed so I will remove